src/libs/storage.js

```diff
@@ -0,0 +1,40 @@
+
+// Storage Variables
+
+const CurrentUser = "CurrentUser"; // System user
+const GoogleUser = "Gapi"; // Google user info, including token
+const UserIsLogged = "isLogged"; // User log status flag
+
+export const Storage = {
```

Comment:
This class is more about session storage than `Storage` in general. I think it could be named better which would distinguish it from other forms of storage, such as local storage, or GCS, for example.

Reply:
ok